src/main.rs

```diff
@@ -127,11 +119,10 @@ async fn main() {
     env_logger::init();
 
     if CONFIG.bot_token == "YOUR_BOT_TOKEN" {
-        eprintln!("Your bot token is still the default, please change it in the config.yaml");
-        return;
+        return eprintln!("Your bot token is still the default, please change it in the config.yaml");
     }
 
-    let intents = GatewayIntents::non_privileged() | GatewayIntents::MESSAGE_CONTENT | GatewayIntents::GUILD_MEMBERS;
+    let intents = GatewayIntents::all() ^ GatewayIntents::GUILD_PRESENCES;
```

Comment:
I prefer the previous version. This one will request more intents if more are added to the API which might break the bot when those intents are privileged and not enabled.